pages/discover.js

```diff
@@ -20,62 +20,61 @@ export default function Discover() {
+            particularly at data synchronization in a distributed set-up. Its main advantages are a
+            single protocol to connect multiple devices, constitutional data security from end to
+            end, and data stores free from centralization constraints.
           </div>
         </div>
 
-        <div className="max-w-md basis-120">
+        <div className="max-w-lg basis-120">
           <h3 className="text-xl sm:text-2xl font-bold text-transparent bg-clip-text bg-gradient-to-r from-blue-500 to-sky-500">
             Secured with market proven algorithms.
           </h3>
           <div className="pt-4">
-            The implementation is independent from dependencies and use basic cryptographic
-            primitives extracted from challenged open-source projects. Moreover, the choice of the
-            algorithms fits to nowadays device capabilities to ensure run time performances that
-            fits to connected objects, or high speed execution of apps.
+            Condensation implementation is self-reliant from software dependencies. It uses
+            cryptographic primitives based on open-source projects continuously challenged by
+            cyber-security communities. The choice of algorithms ensures run time performances that
+            fits common digital devices, IoTs, and high speed apps.
           </div>
         </div>
 
-        <div className="max-w-md basis-120">
+        <div className="max-w-lg basis-120">
           <h3 className="text-xl sm:text-2xl font-bold text-transparent bg-clip-text bg-gradient-to-r from-blue-500 to-sky-500">
             Compliant with regulations.
           </h3>
           <div className="pt-4">
-            Condensation fits to GDPR and other data privacy and protection regulations. Besides the
-            inherent security and the capacity to certify data, it makes a clear statement on who is
-            responsible of the data and where it stored. Thus, data protection and ownership is part
-            of the technology and not of complex mitigations.
+            Condensation is compliant with <a class="underline underline-offset-2 text-blue-500" href="https://gdpr-info.eu/" target="_blank">GDPR</a> and other privacy regulations (contact us for more information). More importantly, data ownership and data storage are clearly defined and
+            managed through the implementation of CDS. This particularity puts the "data sensitivity" at the core of the technology, and no longer 
+            delegated to complex internal measures.
           </div>
         </div>
       </div>
 
       <SubTitle
         content="Guiding principles"
-        title="Condensation puts back the control of data in the hands of its owner while giving the flexibility to build a market ready solution."
+        title="Condensation makes flexible market-ready solutions with the guarantee of data ownership."
```

Comment:
So we said its market ready just before, I would insist on the flexibility and novel approach to ensure data ownership.